Resources/Prototypes/_Sunrise/Surgery/amputation_steps.yml

```
@@ -16,3 +16,70 @@
+      tools:
+        - type: Hemostat
+    - type: Sprite
+      sprite: Objects/Specific/Medical/Surgery/scissors.rsi
+      state: hemostat
+
+- type: entity
+  parent: SurgeryStepBase
+  id: SurgeryStepSawSkull
+  name: Saw Skull
+  components:
+    - type: SurgeryStep
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)